sound/Music.java

```diff
@@ -201,6 +209,13 @@ public void playTune(Tune tune) {
         Iterator<Chord> tuneIt = tune.iterator();
         int line = 0;
         while (tuneIt.hasNext()) {
+            for(int i = 0; i<newRow; i++){
+                row[i] = pixels[i];
```

Comment:
Why is the variable row needed?  That is, why can't the variable pixels be used directly?